### ISSUE TITLE
Configuration Options

### ISSUE THREAD

**Issue by jwillmer** [tidwall/tile38#532]
I tried tile38 in a local docker container and it works as intended. I now like to take the next step and deploy it to our server. However I can't find any documentation of the configuration options. I like to configure (at least) a basic auth option for the client and maybe limit which client (protocols) can connect.

Can anyone provide a link to the configuration options and how to apply them?

**Comment by HkMoyun** [comment]
You can use the local doc window to remotely connect to the server's services and set the configuration .
for example：
First open the doc window, CD enter the directory of tile38_cli . 
Second
![image](https://user-images.githubusercontent.com/24216222/75320281-5101a100-58a9-11ea-8a87-08f7a90ffacd.png)

**Comment by HkMoyun** [comment]
Of course, you can also write code to control tile38 with commands in the program and initialize the configuration . 
If you use Java, this link might be useful for you：https://github.com/HkMoyun/tile38-client

**Comment by jwillmer** [comment]
I'm not asking about how I can interact with the server. I'm asking how I can configure it. I like to set environment variables to define the auth password for example.

**Comment by AIBappa** [comment]
@jwillmer - is this issue still open or you have any resolution for this please?

**Comment by jwillmer** [comment]
I don't have a resolution but it's also not an issue any more. Project ended.
